uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -422,91 +449,16 @@ public Response createSpace(Space space) {
                      space.getOwner(),
                      space.getDefaultGroupId());
 
-        String id = newId();
-        SpaceRequest jobRequest = new SpaceRequest();
+        final String id = newId();
+        final SpaceRequest jobRequest = new SpaceRequest();
         jobRequest.setStatus(JobStatus.ACCEPTED);
         jobRequest.setJobId(id);
         jobRequest.setSpaceName(space.getName());
         jobRequest.setOwner(space.getOwner());
         jobRequest.setDefaultGroupId(space.getDefaultGroupId());
-        jobRequest.setProjects(space.getProjects());
         addAcceptedJobResult(id);
 
-        jobRequestObserver.createOrganizationalUnitRequest(jobRequest);
-
-        return createAcceptedStatusResponse(jobRequest);
-    }
-
-    @POST
-    @Consumes(MediaType.APPLICATION_JSON)
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("/spaces/{spaceName}/")
-    @RolesAllowed({REST_ROLE, REST_PROJECT_ROLE})
-    public Response updateOrganizationalUnit(@PathParam("spaceName") String orgUnitName,
```

Comment:
There should be REST endpoint for updating of space. This REST endpoint is mention in restApi.jsp file in PR into kie-wb-distribtutions

Reply:
I need to double check if we need to remove that. Since the Space does not offer that much that can be edited.